cross/restic/Makefile

```diff
@@ -0,0 +1,28 @@
+PKG_NAME = restic
+PKG_VERS = 0.15.0
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/restic/restic/releases/download/v$(PKG_VERS)
+EXTRACT_PATH = $(WORK_DIR)
```

Comment:
`$(WORK_DIR)` is the default value for `EXTRACT_PATH`, so you can omit this line.

Reply:
Fixed in https://github.com/SynoCommunity/spksrc/pull/5557/commits/1976694e071212c765f57428d6a67276eeb81e7e